content/applications/sales/crm/pipeline/manage_sales_teams.rst

```diff
@@ -0,0 +1,103 @@
+==================
+Manage sales teams
+==================
+
+The *Sales Teams* feature within Odoo's *CRM* app allows for the creation and management of multiple
+sales teams, each with their own assignment rules, invoicing targets, and roster of salespeople.
+
+Create a new sales team
+=======================
+
+To create a new sales team, go to :menuselection:`CRM app --> Configuration --> Sales Teams`, then
+click :guilabel:`New`.
+
+On the blank sales team form, enter a name in the :guilabel:`Sales Team` field.
+
+Next, select a :guilabel:`Team Leader` from the drop-down list.
+
+Set an :guilabel:`Email Alias` to automatically generate a lead/opportunity for this sales team
+whenever a message is sent to that unique email address. Choose whether to accept emails from
+:guilabel:`Everyone`, :guilabel:`Authenticated Partners`, :guilabel:`Followers Only`, or
+:guilabel:`Authenticated Employees`.
+
+.. image:: manage_sales_teams/sales-team-creation.png
+   :align: center
+   :alt: The settings page for a new sales team.
+
+.. note::
+   If the *Sales* app is installed on the database, an :guilabel:`Invoicing Target` field appears on
+   the sales team form. This is the revenue target for the current month. The amount entered in this
+   field is used to populate the invoicing progress bar on the :ref:`sales team dashboard
+   <crm/sales-team-dashboard>`.
+
+Add members to a sales team
+---------------------------
+
+To add team members, click :guilabel:`Add` under the :guilabel:`Members` tab when editing the sales
+team's configuration page. This opens a :guilabel:`Create Sales Team Members` pop-up window.
```

Comment:
I'm actually seeing this titled: "Add: Salespersons" pop-up window. Can you please ensure to confirm the accuracy of this during your revisions? Thanks

Reply:
It's only when I click 'New' from that first "Add: Salespersons" pop-up window that a 'Create Salespersons' pop-up appears...just something that might be worth a quick double-check on your part during revisions.

Reply:
UPDATE: ohhh okay - so you're note admonition below cleared this up. however, since I ran into this issue at this point in the process, it might be best to move that admonition up to right here